src/lib/Containers/Inquiry.tsx

```diff
@@ -0,0 +1,219 @@
+
+import BottomAlignedButton from "../Components/Consignments/Components/BottomAlignedButton"
+
+import ArtworkPreview from "../Components/Inbox/Conversations/Previews/ArtworkPreview"
+import ARSwitchBoard from "../NativeModules/SwitchBoard"
+import { gravityURL } from "../relay/config"
+import { NetworkError } from "../system/errors"
+
+const Container = styled.View`
+  flex: 1
+  flex-direction: column
+`
+const Header = styled.View`
+  alignSelf: stretch
+  margin-top: 10
+  flex-direction: column
+  margin-bottom: 30
+`
+// This is really rubbish, but I basically have to create an equally sized element
+// on the top right, to get the title in the middle
+const PlaceholderView = styled(SmallHeadline)`
+  padding-right: 20
+  color: white
+`
+const TitleView = styled.View`
+  align-self: center
+  align-items: center
+  margin-top: 6
+`
+const HeaderTextContainer = styled.View`
+  flex-direction: row
+  justify-content: space-between
+`
+const CancelButton = styled.TouchableOpacity`padding-left: 20;`
+const Content = styled.View`
+  margin-left: 20
+  margin-right: 20
+`
+const InquiryTextInput = styled.TextInput`
+  font-size: 16
+  margin-top:20
+  font-family: ${fonts["garamond-regular"]}
+`
+const ResponseRate = styled(SmallHeadline)`
+  color: ${colors["yellow-bold"]}
+  marginTop: 5
+`
+const ResponseIndicator = styled.View`
+  width: 8
+  height: 8
+  border-radius: 4
+  margin-top: 5
+  margin-right: 5
+  background-color: ${colors["yellow-bold"]}
+`
+const ResponseRateLine = styled.View`
+  flex: 1
+  flex-direction: row
+  align-items: center
+  min-height: 12
+  margin-top: 5
+`
+
+export class Inquiry extends React.Component<RelayProps, any> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      text: this.props.inquiryArtwork.contact_message,
+      sending: false,
+    }
+  }
+
+  dismissModal() {
+    ARSwitchBoard.dismissModalViewController(this)
+  }
+
+  sendInquiry() {
+    // Using setState to trigger re-render for the button
+    this.setState(previousState => {
+      return { sending: true }
+    })
+    const { Emission } = NativeModules
+    fetch(gravityURL + "/api/v1/me/artwork_inquiry_request", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        "X-ACCESS-TOKEN": Emission.authenticationToken,
+      },
+      body: JSON.stringify({
+        artwork: this.props.inquiryArtwork.id,
+        message: this.state.text,
+      }),
+    })
+      .then(response => {
+        if (response.status >= 200 && response.status < 300) {
+          this.dismissModal()
+        } else {
+          this.setState(previousState => {
+            return { sending: false }
+          })
+          const error = new NetworkError(response.statusText)
+          error.response = response
+          throw error
+        }
+      })
+      .catch(error => {
+        this.setState(previousState => {
+          return { sending: false }
+        })
+        throw error
+      })
```

Comment:
I was considering that, but I'd like to move it all to metaphys next week anyway :)